src/release/issue-triaging.md

```diff
@@ -134,6 +135,10 @@ There are many different labels that can be applied to issues.
     - `S-has-bisection`: A bisection has been performed and the bisection result is convincing.
     - `S-waiting-on-LLVM`: Waiting on upstream LLVM PR or fix.
     - `S-tracking-forever`: The tracking issue is never intended to be closed.
+- [`beta-nominated`]: Tracks changes which need to be [backported to beta].
+- [`stable-nominated`]: Tracks changes which need to be [backported to stable] (in anticipation of a point release).
+- [relnotes]: Changes that should be documented in the release notes of the next release.
```

Comment:
Nit: it has a dual-purpose:

1. It can be used to tag a PR w/ `relnotes`, which will make triagebot create a new relnotes issue.
2. It also marks relnotes issues, so it can be processed by T-release relnotes tooling

Reply:
> It can be used to tag a PR w/ `relnotes`, which will make triagebot create a new relnotes issue.

what is a *relnotes issue*

Reply:
E.g. https://github.com/rust-lang/rust/issues/137132

Reply:
Er I guess I forgor an FCP will also cause a relnotes issue to be created, if it's started on an issue.